cmake/BLTMacros.cmake

```diff
@@ -272,7 +343,22 @@ macro(blt_add_library)
         set(arg_CLEAR_PREFIX TRUE)
     endif()
 
-    if ( arg_SHARED OR ENABLE_SHARED_LIBS )
+
+    #
+    #  cuda support
+    #
+    list(FIND arg_DEPENDS_ON "cuda" check_for_cuda)
```

Comment:
In CMake 3.3, if(<variable|string> IN_LIST <variable>) was added.  I thought we were already past requiring above that version or am i mistaken...  It is much more readable.